README.md

```diff
@@ -15,39 +15,71 @@
   <img src="assets/res.webp"/>
 </p>
 
-# About
-
-This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [Vinceliuice](https://github.com/vinceliuice)
+This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [vinceliuice](https://github.com/vinceliuice)
 
 ## Usage
 
 ### Requirements
 
-- GTK `>=3.20`
+- GTK >=3.20
+- `python3`
 - `gnome-themes-extra` (or `gnome-themes-standard`)
 
-### Installation
+To build the theme, make sure the following packages are installed:
+- `sassc`
+- `inkscape`
+- `optipng`
+
+### Installing the theme manually
 
-1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/).
-2. Move the theme folder to **".themes"** in your home directory. **(~/.themes)** (Skip this step if you are using the AUR package)
-3. Select the downloaded theme via your desktop specific tweaks application (**gnome-tweaks** on Gnome 3+).
+1. Download and extract the theme zip from the [latest release](https://github.com/catppuccin/gtk/releases/latest/).
+2. Move the theme folder to the `~/.local/share/themes` directory (skip this step if you're using the AUR package).
```

Comment:
Not sure that this needs called out? AUR users should use their helper, not the manual steps.
```suggestion
2. Move the theme folder to the `~/.local/share/themes` directory.
```